core/edr-cache-core/src/test/java/org/eclipse/tractusx/edc/edr/core/defaults/InMemoryEndpointDataReferenceCacheTest.java

```diff
@@ -39,7 +40,7 @@ void verify_operations() {
 
```

Comment:
I'm no fan of those "verify-everything-in-one-testmethod" test. I think there should be 1...N test methods for every method of the SuT

Reply:
I Agree but i will work on PosgreSQL edr store next, i would target this refactor there